tests/Avalonia.Layout.UnitTests/LayoutManagerTests.cs

```diff
@@ -238,7 +237,63 @@ public void Invalidating_Child_Remeasures_Parent()
             border.Height = 100;
 
             root.LayoutManager.ExecuteLayoutPass();
-            Assert.Equal(new Size(100, 100), panel.DesiredSize);             
+            Assert.Equal(new Size(100, 100), panel.DesiredSize);
+        }
+
+        [Fact]
+        public void Multiple_Shapes_When_Visibled_Should_Be_Arranged_Properly()
```

Comment:
typo: should be "When_Visible`

Reply:
what about 'When_Made_Visible' as the idea is: visual is non visible, then when it's made visible layout should performed properly?

Reply:
Yes "When_Made_Visible" works 👍